src/seml/experiment/command.py

```diff
@@ -117,7 +130,7 @@ def get_command_from_exp(
                 for k, v in interpolated['config_unresolved'].items()
                 if not k.startswith(SETTINGS.NAMED_CONFIG.PREFIX)
             }
-            named_configs = interpolated[key_named_configs]
+            named_configs = cast(List[str], interpolated[key_named_configs])
```

Comment:
I guess this cast can't be avoided, since we can not dynamically have a correct typing for the input dict for `resolve_config_interpolations` that has `key_named_configs` as a key of type `str`, or can we?

Reply:
I am not sure what a good solution here is while not putting to many assumptions into the utility functions that technically are more general